hooks/check_boilerplate.sh

```diff
@@ -24,7 +24,7 @@ LINES=$(cat "${REF_FILE}" | wc -l | tr -d ' ')
 GO_FILES=$(find . -name "*.go" | grep -v -e "Godeps" -e "third_party" -e "extpoints/extpoints.go")
 
 for FILE in ${GO_FILES}; do
-  DIFFER=$(cat "${FILE}" | sed 's/2015/2014/g' | head "-${LINES}" | diff -q - "${REF_FILE}")
+  DIFFER=$(cat "${FILE}" | sed 's/2015/2014/g;s/2016/2014/g' | head "-${LINES}" | diff -q - "${REF_FILE}")
```

Comment:
Couldn't this be `s/201[0-9]/2014/g`? Then we won't need to change it in 2017 :)